src/sidebar/store/modules/annotations.js

```diff
@@ -441,6 +441,20 @@ const isWaitingToAnchorAnnotations = createSelector(
   annotations => annotations.some(metadata.isWaitingToAnchor)
 );
 
+/**
+ * @template S The type of state held by this store.
+ * @template A the type of actions which may be dispatched by this store.
+ * @typedef AnnotationsStore<S, A>
+ *
+ * // actions
+ * @prop {()=>A} addAnnotations
+ * // TODO: add the rest
+ *
+ * // selectors
+ * @prop {(a: S) => S} annotationCount
```

Comment:
I think this makes sense... takes a "state" and returns a piece of it. Same type for the param and return value.